doc/internal/man7/deprecation.pod

```diff
@@ -0,0 +1,219 @@
+=pod
+
+=head1 NAME
+
+deprecation - Macros used for deprecating stuff and simulate removal
```

Comment:
_stuff_ really?  _things_ is only ever so slightly better.

_functions and macros_ or _symbols_

Reply:
You may have noticed that I have a slightly more relaxed attitude with internal ...  things.  But ok

Reply:
I'd argue that the same rules apply to internal documentation: 1. because it will be read by users and 2. some might be made public over time.